sample_code/ep155/README.md

```diff
@@ -0,0 +1,26 @@
+# [python cffi tutorial (intermediate - advanced)](https://youtu.be/X5irxO5VCHw)
+
+Today I build a sample cffi library, wrapping the `uuid` system module!  cffi is a performant alternative to writing error-prone C extensions and performs well with pypy!
+
+## Interactive examples
+
+### Python
+
+```python
+import uuid
+uuid.UUID(bytes=b'x' * 16)
+
+import uuidcffi
+uuidcffi.generate_uuid()
+```
+
+### Bash
+
+```bash
+virtualenv venv
+. venv/bin/activate
+
+locate uuid.h
+dpkg -L uuid-dev
+pip install .
```

Comment:
This step here might require installing additional libraries into the system, which might be confusing because the video doesn't show that process.